app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminManageStockGridChangeColumnOrderByDragAndDropTest.xml

```diff
@@ -67,7 +67,7 @@
         <grabMultiple selector="{{AdminGridHeaders.columnsNames}}" stepKey="grabColumnsAfterNameColumnMoved" />
         <assertEquals stepKey="assertOrderAfterNameColumnMoved">
             <actualResult type="variable">grabColumnsAfterNameColumnMoved</actualResult>
-            <expectedResult type="array">["Sales Channels", "Name", "ID", "Action"]</expectedResult>
+            <expectedResult type="array">["Sales Channels", "Name", "ID", "Assigned sources", "Action"]</expectedResult>
```

Comment:
looks like these tests are now failing in 2.3-develop branch 
so that this fix should be delivered in separate PR